index.bs

```diff
@@ -485,6 +516,158 @@ spec:csp3; type:grammar; text:base64-value
   failed resource with a different one.
 
   <!-- ####################################################################### -->
+  
+  ## Integrity-Policy ## {#integrity-policy-section}
+  The `Integrity-Policy` and `Integrity-Policy-Report-Only` HTTP headers enable a document to
+  enforce a policy regarding the integrity metadata requirements on all the subresources it
+  loads of certain <a for=request>destinations</a>.
+
+  The headers' value is a <a>Dictionary</a> [[RFC9651]], with every member-value being an
+  <a>inner list</a> of <a>tokens</a>.
+  
+  A <dfn>source</dfn> is a string. The only possible value for it is "`inline`".
+
+  A <dfn>destination</dfn> is a string. The only possible value for it is "`script`".
+
+  An <dfn>integrity policy struct</dfn>, is a <a>struct</a> that contains the following:
+
+    * <dfn>sources</dfn>, a list of <a>source</a>s, Initially empty.
+    * <dfn>blocked destinations</dfn>, a list of <a>destination</a>s, Initially empty.
+    * <dfn>endpoints</dfn>, a <a>list</a> of strings, initially empty.
+
+  When <dfn>processing an integrity policy</dfn>, with |headers| and |header name|, do the following:
+
+  1. Let |integrityPolicy| be a new <a>integrity policy struct</a>.
+  1. Let |dictionary| be the result of <a>getting a structured field value</a> from |headers|
+     given |header name| and "`dictionary`".
+  1. If |dictionary|["`sources`"] does not exist or if its value <a for=list>contains</a> "`inline`",
+     <a for=list>append</a> "`inline`" to |integrityPolicy|'s <a>sources</a>.
+  1. If |dictionary|["`blocked-destinations`"] exists:
+    1. If its value <a for=list>contains</a> "`script`",
+       <a for=list>append</a> "`script`" to |integrityPolicy|'s <a>blocked destinations</a>.
+  1. If |dictionary|["`endpoints`"] <a for=map>exists</a>:
+    1. Set |integrityPolicy|'s <a>endpoints</a> to |dictionary|['endpoints'].
+  1. Return |integrityPolicy|.
+  
+  ### Parse Integrity-Policy headers ### {#parse-integrity-policy-headers-section}
+  To <dfn>parse Integrity-Policy headers</dfn>, given a <a for=/>Response</a> |response|
+  and a <a>policy container</a> |container|, do the following:
+
+  1. Let |headers| be |response|'s <a for=response>header list</a>.
+  1. If |headers| <a for="header list">contains</a> a <a>header</a>
+     whose <a>header name</a> is "`Integrity-Policy`",
+     let |container|'s <a>integrity policy</a> be the result of running
```

Comment:
let or set? It seems like you are overwriting it here. Same below.

Reply:
"set" indeed